### DISCUSSION TITLE
docker容器部署 tvbox json文件里面的地址都是容器ip地址  不是宿主机ip地址

### DISCUSSION THREAD

**Discussion by qppcn** [monlor/docker-xiaoya#194] [Q&A]
![image](https://github.com/monlor/docker-xiaoya/assets/121075185/eec00b8e-04d6-4b0c-8f98-62565c2223af)
docker容器部署 tvbox json文件里面的地址都是容器ip地址  不是宿主机ip地址

**Comment by qppcn** [comment]
1

**Comment by monlor** [comment]
不看文档的么
